--- src/ilm/layout_map.rs
+++ src/ilm/layout_map.rs
@@ -221,6 +221,7 @@
         macro_rules! flush_text {
             () => {
                 if !current_text_blocks.is_empty() {
+                    println!("FLUSH TEXT: {} blocks", current_text_blocks.len());
                     let mut h = 0.0;
                     for p in current_text_blocks.iter() {
                         if let RichBlock::Paragraph(para) = p {
